pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -31,14 +31,15 @@ let
     # but brings high DPI support?
     else wxGTK31.override { withGtk2 = false; };
 
-  pythonPackages = python.pkgs;
   python = python3;
-  wxPython = pythonPackages.wxPython_4_0;
+  wxPython = python.pkgs.wxPython_4_0;
 
 in
 stdenv.mkDerivation rec {
 
   passthru.libraries = callPackages ./libraries.nix versionConfig.libVersion;
+  # a link to this exists as python3.pkgs.[kicad|kicad-unstable]
+  passthru.py = python.pkgs.toPythonModule base;
```

Comment:
please keep the `toPythonModule` only in `python-packages.nix`.

Reply:
this makes it so `kicad-small.py` works

Reply:
Python modules need to be inside `python-packages.nix`, not outside of it.

Reply:
a link to the module exists there
independently defining it there would result in not only duplicating the same snippet at least 4 times
but risk breaking stuff if a pname changes (for example you couldn't do an ad-hoc `(kicad.override {pname ="kicad-testing"; change=otherStuff;};).py` )

Reply:
removing the `kicad.py` attribute would fix those concerns i suppose
but that would feel like a very scattered implementation then

Reply:
can you tell me if there's anything added by `toPythonModule`?
`kicad.base` == `kicad.src` == `kicad.py` == `python3.pkgs.kicad`

i think with the bindings exposed directly (by linking `/lib` (and `/share`?))
having `kicad.py` doesn't make much sense
and `python.pkgs.kicad` makes sense as an end-point for `kicad.base` to avoid pulling in the kicad libraries if you just want the bindings

Reply:
resolved with the new commit? (will be squashed before undrafting)

Reply:
lookup the definition of `toPythonModule`. It adds a `passthru` that sets relevant attributes for it to be usable by `python.buildEnv`/`python.withPackages`.